### PR TITLE
2D Bug Fixes and Optimizations

### PR DESCRIPTION
**Pull request by rainsing**
### Purpose of this PR
2D Bug Fixes and Optimizations. This PR includes changes made in #1097.

---
### Testing status

**Manual Tests**:
- [x] Opened test project + Run graphic tests locally
- [ ] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in developer mode, you have a button at end of resources that check the paths)

**Automated Tests**: NA
---
### Comments to reviewers
NA

### CODE REVIEW

File: com.unity.render-pipelines.universal/CHANGELOG.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - The 2D Renderer now supports camera stacking.
 - 2D shaders now use half-precision floats whenever precise results are not necessary.
 - Removed the ETC1_EXTERNAL_ALPHA variant from Shader Graph Sprite shaders.
-- Eliminated GC allocations caused by the 2D Renderer.
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/CHANGELOG.md:72]: Is this change intentional? Seems like it's overriding another entry in the log.

Comment (rainsing) [reply com.unity.render-pipelines.universal/CHANGELOG.md:72]: Yes. It was a duplicate entry.
